CLAUDE.md

```
@@ -141,13 +181,20 @@ npx vitest run apps/api      # API tests only (from repo root)
+**`apps/web` exception**: The Next.js workspace must be run from its own directory — the `@/` alias is resolved by `apps/web/vitest.config.ts`, not the root runner:
+
+```bash
+cd apps/web && pnpm test     # web tests only (vitest passWithNoTests)
+```
+
 ### Vitest Configuration
 
-Both workspaces use `mockReset: true` — the single most important config detail:
+All three workspaces use `mockReset: true` — the single most important config detail:
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)